### PR TITLE
feat: add config and command to toggle off code lens

### PR DESCRIPTION
**Pull request by Hellebore**
Review with https://github.com/sourcery-ai/core/pull/2831

## Checklist

- [ ] If `package.json` or `yarn.lock` have changed, then test the VSIX built by `yarn run vsce package` works from a direct install

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -234,6 +234,11 @@
 				"title": "Ask Sourcery",
 				"category": "Sourcery"
 			},
+			{
+				"command": "sourcery.chat.toggleCodeLens",
+				"title": "Toggle Sourcery Code Lens",
+				"category": "Sourcery"
```
Comment (brendanator) [inline package.json:240]: **🎯 suggestion:** ‏Add a when clause here too?

File: package.json [hunk 2]
```diff
@@ -234,6 +234,11 @@
 				"title": "Ask Sourcery",
 				"category": "Sourcery"
 			},
+			{
+				"command": "sourcery.chat.toggleCodeLens",
+				"title": "Toggle Sourcery Code Lens",
```
Comment (brendanator) [inline package.json:239]: Include `coding assistant` in the title?

**Review by brendanator — APPROVED** [review]
I feel like this should be a configuration option that appears in the settings like these options and gets written into the user config file
image

Maybe that's a follow up change?

File: package.json (hunk 1 above)
Comment (Hellebore) [reply package.json:240]: I don't think you add `when` clauses when defining the commands?

**Pull request merged**